src/meta/model_v2/src/function.rs

```diff
@@ -41,6 +41,7 @@ pub struct Model {
     pub language: String,
     pub link: String,
     pub identifier: String,
+    pub body: String,
```

Comment:
We can make it an Option.

Reply:
Does `Model` need the value of `body`?
If not I'll just insert `None` whenever this field is needed.

Reply:
Yes, it's necessary.